client/src/battle/battle_simulation.rs

```diff
@@ -1032,6 +1034,28 @@ impl BattleSimulation {
 
         sorted_entities.sort_by_key(|(_, key)| *key);
 
+        if let Ok((entity, character)) =
+            (self.entities).query_one_mut::<(&Entity, &Character)>(self.local_player_id.into())
+        {
+            let offset: Vec2 = entity.corrected_offset(self.perspective_flipped);
+            let tile_center =
+                (self.field).calc_tile_center((entity.x, entity.y), self.perspective_flipped);
+            let x_difference = 2.0;
+            let mut position = vec2(
+                tile_center.x + offset.x,
+                tile_center.y - entity.height - 16.0,
+            );
+            for card in &character.cards {
+                let blank_card = Card {
+                    package_id: card.package_id.clone(),
+                    code: String::new(),
+                };
+                blank_card.draw_icon(game_io, &mut sprite_queue, position);
+                position.x += x_difference;
```

Comment:
Little trick, the Vec2s in glam (the math library we're using) implements the add operation for Vec2 + f32. So this could be shortened to just `position += x_difference;`